src/cli.ts

```diff
@@ -57,6 +59,8 @@ export default function run(): void {
       transforms: commander.transforms.split(","),
       enableLegacyTypeScriptModuleInterop: commander.enableLegacyTypescriptModuleInterop,
       enableLegacyBabel5ModuleInterop: commander.enableLegacyBabel5ModuleInterop,
+      jsxPragma: commander.jsxPragma || "React.createElement",
+      jsxFragmentPragma: command.jsxFragmentPragma || "React.Fragment",
```

Comment:
Oh, build failure actually was caused by this commit. Just a typo, `command` -> `commander.`
```suggestion
      jsxFragmentPragma: commander.jsxFragmentPragma || "React.Fragment",
```

Reply:
Just assigned the PR to myself so I can apply the suggestion and see if it fixes things!

Reply:
Ah, nevermind, I can't push to your fork. Would be good for you to apply when you get a chance, hopefully will just fix things!

<img width="542" alt="screen shot 2018-10-20 at 2 17 58 pm" src="https://user-images.githubusercontent.com/211605/47260572-fd8c7100-d472-11e8-9a56-d9e9e2c46c9c.png">